core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the preferences page, for use in Protractor
+ * tests.
+ */
+
+ var PreferencesPage = function() {
+ 	var USER_PREFERENCES_URL = '/preferences';
```

Comment:
Please use spaces instead of tabs. (Always check your file in GitHub after submitting to ensure that the changes are what you want them to be: see https://github.com/oppia/oppia/pull/4103/files .)

Reply:
Will take care of this from now on.